spec/book_spec.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/book'
+
+describe Books do
+  books = [{ title: 'managment101', author: 'rich' }, { title: 'cooking101', author: 'fat'}]
```

Comment:
Metrics/LineLength: Line is too long. [92/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)